idea/resources/intentionDescriptions/ReplaceMathMaxWithCoerceAtLeastIntention/description.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+    This intention detects 'Math.max' calls that can be safely replaced with 'coerceAtLeast'
```

Comment:
Moreover, the word 'safe' is not needed here. Looks like replacement is possible in any case.